src/Command/SqlGeneratorCommand.php

```diff
@@ -0,0 +1,229 @@
+        }
+
+        return 0;
+    }
+
+    public function generateSqlForBean($className) : string
+    {
+        $bean = new \ReflectionClass($className);
+
+        $toReturn = 'CREATE TABLE `' . \Infinityloop\Utils\CaseConverter::toSnakeCase($bean->getShortName()) . '`(' . \PHP_EOL;
+        $foreignKeys = '';
+        $data = [];
+
+        foreach ($bean->getProperties(\ReflectionProperty::IS_PUBLIC) as $property) {
+            if (!$property->getType() instanceof \ReflectionType) {
+                continue;
+            }
+
+            $data[] = $this->getData($property);
+            $foreignKeys .= $this->getForeignKey($property);
+        }
+
+        $toReturn .= $this->buildTable($data);
+
+        $toReturn .= ($foreignKeys === '' ? '' : \PHP_EOL) . $foreignKeys;
+
+        $toReturn .= ')' . \PHP_EOL;
+        $toReturn .= self::INDENTATION . 'CHARSET = `utf8mb4`' . \PHP_EOL;
+        $toReturn .= self::INDENTATION . 'COLLATE `utf8mb4_general_ci`;' . \PHP_EOL . \PHP_EOL;
+
+        return $toReturn;
+    }
+
+    private function buildTable(array $data) : string
+    {
+        $longestNameLength = $this->getLongestByType($data, 'name');
+        $longestDataTypeLength = $this->getLongestByType($data, 'dataType');
+        $toReturn = '';
+
+        foreach ($data as $row) {
+            $nameLength = \mb_strlen($row['name']);
+            $dataTypeLength = \mb_strlen($row['dataType']);
+            $toReturn .= self::INDENTATION . $row['name'] . \str_repeat(' ', $longestNameLength - $nameLength + 1);
+            $toReturn .= $this->buildDataType($row, $longestDataTypeLength, $dataTypeLength);
+            $toReturn .= $row['notNull'] === true ? ' NOT NULL' : '';
+            $toReturn .= $row['default'];
+            $toReturn .= $this->buildAutoIncrement($row['name']);
+            $toReturn .= ',' . \PHP_EOL;
+        }
+
+        return $toReturn;
+    }
+
+    private function buildAutoIncrement(string $name) : string
+    {
+        return $name === '`id`' ? ' AUTOINCREMENT' : '';
+    }
+
+    private function buildDataType(array $row, int $longestDataTypeLenght, int $dataTypeLenght) : string
+    {
+        if ($row['notNull'] === false && $row['default'] === '') {
+            return $row['dataType'];
+        }
+
+        if ($row['notNull'] === false && $row['default'] !== '') {
+            return $row['dataType'] . \str_repeat(' ', $longestDataTypeLenght - $dataTypeLenght + 9);
+        }
+
+        return $row['dataType'] . \str_repeat(' ', $longestDataTypeLenght - $dataTypeLenght);
+    }
+
+    private function getLongestByType(array $data, string $type) : int
+    {
+        $maxLength = 0;
+
+        foreach ($data as $row) {
+            $length = \mb_strlen($row[$type]);
+
+            if ($length > $maxLength) {
+                $maxLength = $length;
+            }
+        }
+
+        return $maxLength;
+    }
+
+    private function getData(\ReflectionProperty $property) : array
```

Comment:
Unnecessary function - used only once - and it also doesn't contain that much code.